src/main/java/com/google/collegeplanner/data/Course.java

```diff
@@ -0,0 +1,201 @@
+   * methods students can opt for. Example, regular, pass-fail.
+   */
+  private String[] gradingMethod;
+  /*
+   * coreqs represents the corequsite classes for this
+   * course. Example, BIO106 is the lab for BIO101 and
+   * must be taken together
+   */
+  private String coreqs;
+  /*
+   * prereqs represents the prerequsite classes for
+   * this course. Example, MATH101, MATH201, and
+   * either MATH105 or MATH115.
+   */
+  private String prereqs;
+  /*
+   * restrictions represents the restrictions placed on the
+   * registration of this course. Example, This
+   * class is only avaiable to incoming Freshman.
+   */
+  private String restrictions;
+  /*
+   * additionalInfo represents the additional information
+   * for this course found on the course catalog.
+   */
+  private String additionalInfo;
+  /*
+   *creditGrantedFor represents the classes this course
+   * grants credit for. Example, THET285 grants credit for COMM107
+   */
+  private String creditGrantedFor;
+
+  public Course(String courseId, String name, String semester, int credits, String deptId,
+      String description, String[] gradingMethod, String coreqs, String prereqs,
+      String restrictions, String additionalInfo, String creditGrantedFor) {
+    this.courseId = courseId;
+    this.name = name;
+    this.semester = semester;
+    this.credits = credits;
+    this.deptId = deptId;
+    this.description = description;
+    this.gradingMethod = gradingMethod;
+    this.coreqs = coreqs;
+    this.prereqs = prereqs;
+    this.restrictions = restrictions;
+    this.additionalInfo = additionalInfo;
+    this.creditGrantedFor = creditGrantedFor;
+  }
+
+  public Course(JSONObject json) {
+    this((String) json.get("course_id"), (String) json.get("name"), (String) json.get("semester"),
+        (int) json.get("credits"), (String) json.get("dept_id"), (String) json.get("description"),
+        null, (String) json.get("coreqs"), (String) json.get("prereqs"),
+        (String) json.get("restrictions"), (String) json.get("additional_info"),
+        (String) json.get("credit_granted_for"));
+
+    this.gradingMethod = (String[]) ((JSONArray) json.get("grading_method")).toArray();
+  }
+
+  /*
+   * Returns the season of the semester
+   * The format of the semester String is the four digit year followed by
+   * the two digit start month.
+   */
+  public String getSemesterSeason() {
+    if (semester.length() != 6) {
+      return "Invalid Semester Format";
+    }
+    // startMonth stores the last two digits of the semester String as an int
+    int startMonth = Integer.parseInt(semester) % 100;
+
+    switch (startMonth) {
+      case 1:
+      case 2:
+        return "Spring";
+      case 6:
+        return "Summer I";
+      case 7:
+        return "Summer II";
+      case 8:
+      case 9:
+        return "Fall";
+      case 12:
+        return "Winter";
+      default:
+        return "Invalid Month";
```

Comment:
you're returning two types of errors here as two different strings. Should this throw an exception instead? Otherwise callers will need to check the value of the returned string before they can use it.